README.md

```diff
@@ -59,6 +59,7 @@ Copy the generated secrets in to your `.env` (described below)
 Your `.env` file should contain the following:
 
 ```bash
+BATTLESNAKEIO_DOMAIN=play.battlesnake.io
```

Comment:
it won't be this on localhost will it?

Reply:
ah, true. While getting my env up to speed (a couple commits have come in since I last touched this) I needed to set that env, but didn't really experiment with what it was for. What do you set that for? and is it a sane default? or should I use a `...` ellipsis in the readme?

Reply:
it's not needed for local dev, you run the local server with `ENV=local ./manage.py runserver`

Reply:
all that's really required is 
```
BATTLESNAKEIO_GITHUB_CLIENT_ID=...
BATTLESNAKEIO_GITHUB_CLIENT_SECRET=...
```

Reply:
I echo what @dlsteuer said, you only need those two at a minimum, and run the manage command with the supplied args in the README.

Reply:
yeah, `ENV=local` is critical otherwise it runs in production mode